packages/gasket-plugin-fastify/lib/utils.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+const logLevels = {
+  debug: console.debug,
+  error: console.error,
+  info: console.info,
+  warn: console.warn,
+  fatal: console.error,
+  trace: console.debug
+};
+
+/**
+ * A helper function to ensure that all the required log levels
+ * for a Fastify server are present.
+ * @type {import('./index').alignLogger}
+ */
+function alignLogger(logger) {
+  const fastifyLogger = logger;
+  ['fatal', 'trace', 'debug', 'error', 'warn', 'info'].map(level => {
+    if (!logger[level]) {
+      fastifyLogger[level] = logLevels[level];
```

Comment:
You will want to fallback to the logger's methods, not the console's.

`debug`, `error`, `info`, and `warn` are guaranteed ([verify code](https://github.com/godaddy/gasket/blob/v7/packages/gasket-plugin-logger/lib/index.js#L15-L21)) to be on the logger.

```
const fallbackMap = {
  fatal: 'error',
  trace: 'debug'
}
```

```suggestion
      fastifyLogger[level] = logger[fallbackMap[level]];
```

Reply:
Sounds good! Just made that update.